src/main/java/com/thealgorithms/others/SpiralMatrixII.java

```diff
@@ -0,0 +1,49 @@
+package com.thealgorithms.others;
+
+import java.util.Arrays;
```

Comment:
This import is not used. Is that right?

```suggestion
```

Reply:
ohh yup ... Sorry i would have missed it